Dangerfile

```diff
@@ -68,14 +79,14 @@ unless has_changelog || declared_trivial
   pr_author = github.pr_author
   pr_author_url = "https://github.com/#{pr_author}"
 
-  need_fixes = fail("Please include a CHANGELOG entry to credit your work.  \nYou can find it at [CHANGELOG.md](#{repo_url}/blob/master/CHANGELOG.md).")
+  need_fixes = fail("Please include a CHANGELOG entry to credit your work.  \nYou can find it at [CHANGELOG.md](#{repo_url}/blob/develop/CHANGELOG.md).")
```

Comment:
Is this because we're assuming all changelog changes will happen on develop?

Reply:
Yeah I thought about writing `stable` here but figured that since the Danger warning is for contribution PRs (so, targeting `develop`), it made more sense to link to the `develop` version of the CHANGELOG.

An even more accurate solution would be to generate the link by using the branch name of the current PR in that link. Not sure it's worth it?

Reply:
Actually that's not a bad idea. Just changed it: 9bdc98c3e593fc30424a7eb1d3d3cbe55504dbea

Reply:
@Github, we need a rubber duck emoji...🦆

Reply:
![image](https://user-images.githubusercontent.com/216089/85065215-1780d880-b1ad-11ea-91fb-81569a9ec854.png)